provisioners/windows/azure/azure-bootstrap.ps1

```diff
@@ -316,6 +319,20 @@ If (($stage -eq 'setup') -or ($stage -eq 'inprogress')){
     exit 0
 }
 If ($stage -eq 'complete') {
-    Write-Log -message  ('{0} ::Bootstrapping appears complete'  -f $($MyInvocation.MyCommand.Name)) -severity 'DEBUG'
-    exit 0
+    Write-Log -message  ('{0} ::Bootstrapping appears complete' -f $($MyInvocation.MyCommand.Name)) -severity 'DEBUG'
+	Write-Log -message  ('{0} ::Beginning Pester tests' -f $($MyInvocation.MyCommand.Name)) -severity 'DEBUG'
+	## Import module to bootstrap pester and run tests
+	Import-Module "$env:systemdrive\ronin\provisioners\windows\modules\Bootstrap\Bootstrap.psm1"
+	Write-Output ("Processing {0}" -f $ENV:COMPUTERNAME)
+	## Remove old version of pester and install new version if not already running 5
+	if ((Get-Module -Name Pester -ListAvailable).version.major -ne 5) {
```

Comment:
Let's set an array on OSes we want to run to the Pester test on, and then only run it on those OSes. Eventually we will run it on all OSes.

Reply:
Resolved [here](https://github.com/mozilla-platform-ops/ronin_puppet/pull/418/commits/cc60946c266817a2eb208cf5d4f0fc838c153e0d)